tools/deb/istio-iptables.sh

```diff
@@ -101,46 +109,47 @@ if [ -z "${ENVOY_UID:-}" ]; then
   ENVOY_UID=${ENVOY_UID},0
 fi
 
-# Remove the old chains, to generate new configs.
-iptables -t nat -D PREROUTING -p tcp -j ISTIO_INBOUND 2>/dev/null
-iptables -t nat -D OUTPUT -p tcp -j ISTIO_OUTPUT 2>/dev/null
-
-# Flush and delete the istio chains
-iptables -t nat -F ISTIO_OUTPUT 2>/dev/null
-iptables -t nat -X ISTIO_OUTPUT 2>/dev/null
-iptables -t nat -F ISTIO_INBOUND 2>/dev/null
-iptables -t nat -X ISTIO_INBOUND 2>/dev/null
-iptables -t nat -F ISTIO_REDIRECT 2>/dev/null
-iptables -t nat -X ISTIO_REDIRECT 2>/dev/null
-
 if [ "${1:-}" = "clean" ]; then
+  # TODO(nmittler): Had to move all of this to the 'clean' block since it was breaking the sidecar deployment.
+  # Remove the old chains, to generate new configs.
+  iptables -t nat -D PREROUTING -p tcp -j ISTIO_REDIRECT 2>/dev/null
+  iptables -t nat -D OUTPUT -p tcp -j ISTIO_OUTPUT 2>/dev/null
+
+  # Flush and delete the istio chains
+  iptables -t nat -F ISTIO_OUTPUT 2>/dev/null
+  iptables -t nat -X ISTIO_OUTPUT 2>/dev/null
+  iptables -t nat -F ISTIO_INBOUND 2>/dev/null
+  iptables -t nat -X ISTIO_INBOUND 2>/dev/null
+  iptables -t nat -F ISTIO_REDIRECT 2>/dev/null
+  iptables -t nat -X ISTIO_REDIRECT 2>/dev/null
+
   # Only cleanup, don't add new rules.
   exit 0
 fi
 
 # Create a new chain for redirecting inbound traffic to the common Envoy port.
-# In the ISTIO_INBOUND and ISTIO_OUTBOUND chains, '-j RETURN' bypasses Envoy
-# and '-j ISTIO_REDIRECT' redirects to Envoy.
 iptables -t nat -N ISTIO_REDIRECT
 iptables -t nat -A ISTIO_REDIRECT -p tcp -j REDIRECT --to-port ${ENVOY_PORT:-15001}
 
 # Handling of inbound ports. Traffic will be redirected to Envoy, which will process and forward
 # to the local service. If not set, no inbound port will be intercepted by istio iptables.
-if [ -n "${ISTIO_INBOUND_PORTS:-}" ]; then
```

Comment:
Can we keep the old name ? It is used in the env variables, no reason to change it. 

Also on a VM - there are all kind of inbound ports, better to be clea.